package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
```

Comment:
this file should not exist; we have separate `package.json` and corresponding `package-lock.json` files for each of our 3 packages.